### ISSUE TITLE
Missing matches

### ISSUE THREAD

**Issue by nurgasemetey** [openfootball/italy#11]
Hello,

Firstly, thanks for this repo.

I am just wondering why results of these matches are missing

https://github.com/openfootball/italy/blob/e978b367d0ecf7e548fe4eff1e955b92b7ce1657/2019-20/1-seriea.txt#L449-L615

**Comment by geraldb** [comment]
Good catch.  There's no magic.   The `2019-20/1-seriea.txt` datafile needs to get updated :-).   I only run (weekly) auto-updates for the new season 2020/21 for now.   Anyways, I try to update 2019-20 in the next days (hopefully today or tomorrow). Thanks again for reporting.
